src/v/pandaproxy/schema_registry/requests/post_subject_versions.h

```diff
@@ -107,6 +113,28 @@ class post_subject_versions_request_handler
         return false;
     }
 
+    bool Null() {
```

Comment:
q: how exactly should i read this off? like "can this field be null?". having trouble finding clear info in rjson docs.

Reply:
> q: how exactly should i read this off? like "can this field be null?". having trouble finding clear info in rjson docs.

"You are a SAX-ish handler, deal with this `null`."
or
"I am a SAX-ish handler, I'm dealing with this `null`."

Reply:
Note the inheritance from the base class [json::base_handler<Encoding>](https://github.com/BenPope/redpanda/blob/55bd576e345459a4468c357962bdb2541e600f31/src/v/pandaproxy/schema_registry/requests/post_subject_versions.h#L32), which switches the [default](https://github.com/BenPope/redpanda/blob/55bd576e345459a4468c357962bdb2541e600f31/src/v/pandaproxy/json/rjson_parse.h#L29) from returning true if unknown to returning `false` if unknown, for purposes of being overly strict in the sense of "If I don't know what this is, I shouldn't handle it."